diff --git a/summon-arm-toolchain b/summon-arm-toolchain
index ff196e8..7cb847b 100755
--- a/summon-arm-toolchain
+++ b/summon-arm-toolchain
@@ -19,8 +19,8 @@ set -e
 # You probably want to customize those
 ##############################################################################
 TARGET=arm-none-eabi		# Or: TARGET=arm-elf
-PREFIX=${HOME}/sat	# Install location of your final toolchain
 DARWIN_OPT_PATH=/opt/local	# Path in which MacPorts or Fink is installed
+PREFIX=${DARWIN_OPT_PATH}	# Install location of your final toolchain
 # Set to 'sudo' if you need superuser privileges while installing
 SUDO=
 # Set to 1 to be quieter while running
@@ -30,7 +30,7 @@ USE_LINARO=1
 # Set to 1 to enable building of OpenOCD
 OOCD_EN=1
 # Set to 1 to build libstm32 provided by ST
-LIBSTM32_EN=0
+LIBSTM32_EN=1
 # Set to 1 to build libopenstm32 an open source library for stm32
 LIBOPENSTM32_EN=1
 # Make the gcc default to Cortex-M3
