Index: paparazzi-jsbsim-1.5/autogen.sh
===================================================================
--- paparazzi-jsbsim-1.5.orig/autogen.sh	2015-04-24 15:09:40.173010883 +0200
+++ paparazzi-jsbsim-1.5/autogen.sh	2015-04-24 15:11:22.097013311 +0200
@@ -37,11 +37,11 @@
   DIE=1
 }

-LIBTOOL=`(which glibtool || which libtool || echo "libtool") 2> /dev/null`
+LIBTOOLIZE=`(which glibtoolize || which libtoolize || echo "libtoolize") 2> /dev/null`
 (grep "^AC_PROG_LIBTOOL" configure.in >/dev/null) && {
-  ($LIBTOOL --version) < /dev/null > /dev/null 2>&1 || {
+  ($LIBTOOLIZE --version) < /dev/null > /dev/null 2>&1 || {
     echo
-    echo "**Error**: You must have \`libtool' installed to compile $PKG_NAME."
+    echo "**Error**: You must have \`libtoolize' installed to compile $PKG_NAME."
     echo "Get ftp://ftp.gnu.org/pub/gnu/libtool-1.2d.tar.gz"
     echo "(or a newer version if it is available)"
     DIE=1
