diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b268d547..2ab97592 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -121,6 +121,8 @@ endif()
 
 add_executable(JSBSim JSBSim.cpp)
 target_link_libraries(JSBSim libJSBSim)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/JSBSim.pc.in ${CMAKE_CURRENT_BINARY_DIR}/JSBSim.pc @ONLY)
 
 install(TARGETS JSBSim RUNTIME DESTINATION bin COMPONENT runtime)
 install(FILES ${HEADERS} DESTINATION include/JSBSim COMPONENT devel)
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/JSBSim.pc DESTINATION lib${LIB_SUFFIX}/pkgconfig COMPONENT devel)
diff --git a/src/JSBSim.pc.in b/src/JSBSim.pc.in
new file mode 100644
index 00000000..d92c6fb4
--- /dev/null
+++ b/src/JSBSim.pc.in
@@ -0,0 +1,9 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@/bin
+libdir=@CMAKE_INSTALL_PREFIX@/lib@LIB_SUFFIX@
+includedir=@CMAKE_INSTALL_PREFIX@/include
+Name: JSBSim
+Description: JSBSim (Paparazzi version)
+Version: @PROJECT_VERSION@
+Libs: -L${libdir} -lJSBSim
+Cflags: -I${includedir}/JSBSim
