--- src/Makefile.orig	2011-01-31 22:03:01.000000000 +1100
+++ src/Makefile	2011-01-31 22:03:20.000000000 +1100
@@ -25,7 +25,7 @@
 CC=${compiler_prefix}gcc
 AR=${compiler_prefix}ar
 AS=${compiler_prefix}as
-CFLAGS=-I../include -fno-common -Os -g -mcpu=cortex-m3 -mthumb -ffunction-sections -fdata-sections
+CFLAGS=-I../include -fno-common -g -mcpu=cortex-m3 -mthumb -ffunction-sections -fdata-sections
 ARFLAGS=rcsv
 ASFLAGS=-ahls -mapcs-32
 
