--- src/Makefile.orig	2011-01-15 11:26:41.000000000 +1100
+++ src/Makefile	2011-01-15 11:28:01.000000000 +1100
@@ -25,7 +25,7 @@
 CC=${compiler_prefix}gcc
 AR=${compiler_prefix}ar
 AS=${compiler_prefix}as
-CFLAGS=-I../include -fno-common -Os -g -mcpu=cortex-m3 -mthumb -ffunction-sections -fdata-sections
+CFLAGS=-I../include -Wall -fno-common -Os -g -mcpu=cortex-m3 -mthumb -ffunction-sections -fdata-sections -D'assert_param(_cond)={}'
 ARFLAGS=rcsv
 ASFLAGS=-ahls -mapcs-32
 
