--- /Users/eric/gcc-linaro-4.5-2011.01-1 2/configure.ac.orig	2011-02-16 19:05:59.000000000 +1030
+++ /Users/eric/gcc-linaro-4.5-2011.01-1 2/configure.ac	2011-02-16 21:07:30.000000000 +1030
@@ -590,7 +590,6 @@
     ;;
   arm-*-elf* | strongarm-*-elf* | xscale-*-elf* | arm*-*-eabi* )
     noconfigdirs="$noconfigdirs target-libffi target-qthreads"
-    libgloss_dir=arm
     ;;
   arm*-*-linux-gnueabi)
     noconfigdirs="$noconfigdirs target-qthreads"
